README.md

```diff
@@ -78,6 +78,35 @@ For further info refer to [Source Installation on Ubuntu](https://maliput.readth
 
 It is recommended to follow the guidelines for setting up a development workspace as described [here](https://maliput.readthedocs.io/en/latest/developer_setup.html).
 
+## Profiling maliput
+
+`maliput` is able to run a profiler for evaluating performance.
+It is implemented via `ign-common3`'s `profiler` component.
+
+In order to avoid any performance drop, this is disabled by default and the binaries aren't distributed with the profiler set up.
```

Comment:
Main reason could not be the performance drop but probably to keep maliput dependency chain as reduced as possible.